R/DataSet.R

```diff
@@ -409,13 +409,23 @@ get_FV_overview.DataSet <- function(ds, ...) {
 #' @export
 #'
 get_RT_overview.DataSet <- function(ds, ...) {
-  data <- ds$RT
-  runs <- ncol(data)
-  budget <- max(attr(ds, 'maxRT'))
-
-  min_rt <- min(data, na.rm = T)
-  max_rt <- max(data, na.rm = T)
-
+  
+  if(!is.null(attr(ds,"format")) && attr(ds,"format") == NEVERGRAD){
```

Comment:
Doesn't it make more sense to only have ds$FV for nevergrad, since this is the variable used in the fixed-budget case? I think I made a mistake when creating these overview functions, and mixed up FV and RT, since for all other functions it is consistently using FV for fixed-budget and RT for fixed-target